src/status_im/ethereum/macros.clj

```diff
@@ -4,7 +4,7 @@
 
 (defn token-icon-path
   [network symbol]
-  (let [s     (str "./resources/images/tokens/" (name network) "/" (name symbol) ".png")
+  (let [s     (str "../resources/images/tokens/" (name network) "/" (name symbol) ".png")
```

Comment:
just to make sure, this change is intentional?

Reply:
yes path is different